added vyper for vrf, price feeds, and keepers (#890)
smartcontractkit/documentation: docs/Using Randomness/get-a-random-number.md

1. Under your subscription details, click **Cancel subscription**. A field opens asking which wallet address you want to send the remaining funds to.

1. Enter your wallet address and click **Cancel subscription**. MetaMask opens and asks you to confirm the transaction. After you approve the transaction, Chainlink VRF closes your subscription account and sends the remaining LINK to your wallet.

> ## Vyper Example

Reviewer: ```suggestion
## Vyper example
```

> You'd need to import the `VRFCoordinatorV2` vyper interface, you can find it  [here](https://github.com/smartcontractkit/apeworx-starter-kit/blob/main/contracts/interfaces/VRFCoordinatorV2.vy).

Reviewer 2: ```suggestion
You must import the `VRFCoordinatorV2` vyper interface. You can find it  [here](https://github.com/smartcontractkit/apeworx-starter-kit/blob/main/contracts/interfaces/VRFCoordinatorV2.vy).
```

Reviewer 2: Also, is Vyper captialized? I wasn't too sure

Reviewer 3: @ritika-b  please approve :)
